tests/molecules/He_param/CMakeLists.txt

```diff
@@ -24,6 +24,41 @@ if(NOT QMC_CUDA)
     0
     SCALAR_VALUES
     HE_BSPLINE_PARAM)
+
+
+  #  Test loading from variational parameter file
+  check_python_reqs(h5py he_param_h5 add_h5_tests)
+  if (add_h5_tests)
+    if (QMC_COMPLEX)
+      set(complex_flag "--complex")
+    else()
+      set(complex_flag)
+    endif()
+    set(SDIR "${qmcpack_SOURCE_DIR}/tests/molecules/He_param/")
+    # Normally the directory is created by qmc_run_and_check_custom_scalar,
+    # but we need to write a file there before that runs
+    set(TDIR "${qmcpack_BINARY_DIR}/tests/molecules/He_param/He_param_grad_load-1-16/")
+    file(MAKE_DIRECTORY ${TDIR})
+    execute_process(COMMAND ${qmcpack_SOURCE_DIR}/tests/molecules/He_param/convert_vp_format.py ${SDIR}/he_vp_opt.txt ${complex_flag} -o ${TDIR}/he_vp_opt.h5)
+
+    list(APPEND HE_BSPLINE_OPT_PARAM jud_0  0.00000124  0.0014) # scalar name, value, error
+    list(APPEND HE_BSPLINE_OPT_PARAM jud_1 -0.000273    0.00097)
+    list(APPEND HE_BSPLINE_OPT_PARAM jud_2 -0.000181    0.00082)
+    list(APPEND HE_BSPLINE_OPT_PARAM jud_3  0.0004463   0.000058)
+
+    qmc_run_and_check_custom_scalar(
+      BASE_NAME He_param_grad_load
+      BASE_DIR "${qmcpack_SOURCE_DIR}/tests/molecules/He_param"
+      PREFIX He_param_grad_load.param
+      INPUT_FILE He_param_grad_load.xml
+      PROCS 1
+      THREADS 16
+      SERIES 0
+      SCALAR_VALUES HE_BSPLINE_OPT_PARAM)
+
```

Comment:
I'm a bit lost. Where is h5diff being triggered?

Reply:
The test compares the computed parameter gradients.    So h5diff is not used.
The test sets up the wavefunction and loads from the VP file, computes parameter gradients, and then compares those gradients to expected values.

Reply:
There's no test that runs the optimizer and outputs VP files, then runs a comparison on the VP files.  Maybe there should be.

Reply:
I see. txt is converted to vp.h5 and then loaded by QMCPACK.